tests/dft-custom-mgga/input.dat

```diff
@@ -0,0 +1,111 @@
+# DFT (meta-GGA) test of implementations in: mgga_superfuncs.py
+
+## Programs/References
+# ORCA v.4.0.1: cc-pVTZ, RI-MP2, no FC, GRID7
+#-> TPSS
+# [revTPSS differs from Gaussian]
+#
+# Gaussian 09: cc-pVDZ 5D SP Int(Grid=99770)
+#-> revTPSS
+#-> M11L
+## Notes
+# M11L only up to 4 decimals even for Ne with fine grid
+
+psi4.core.set_output_file("output.dat", False)
+
+molecule ne {
+  0 1
+  Ne
+}
+
+set df_scf_guess false
+set basis cc-pvdz
+set reference rks
+set dft_radial_points 99
+set dft_spherical_points 770 
+set scf_type direct
+
+ 
+edft = energy('tpss')
+compare_values(-128.910940991792, edft, 6, 'Ne: TPSS')   #TEST
+clean()
+
+edft = energy('revtpss')
+# ORCA: (large difference?)
+#compare_values(-128.850163295495, edft, 6, 'Ne: revTPSS')   #TEST
+# G09 values
+compare_values(-128.854622168, edft, 5, 'Ne: revTPSS')   #TEST
+clean()
+
+# Gaussian09
+edft = energy('m11-l')
+compare_values(-128.873042062, edft, 4, 'Ne: M11-L')   #TEST
+clean()
+#  
+
+#====== former dft-dldf test ==========
+#! Dispersionless density functional (dlDF+D) internal match to Psi4
+#! Extensive testing has been done to match supplemental info of
+#! Szalewicz et. al., Phys. Rev. Lett., 103, 263201 (2009) and Szalewicz et. al., J. Phys. Chem. Lett., 1, 550-555 (2010)
+
+molecule water {
+0 1
+O  -1.551007  -0.114520   0.000000
+H  -1.934259   0.762503   0.000000
+H  -0.599677   0.040712   0.000000
+--
+0 1
+O   1.350625   0.111469   0.000000
+H   1.680398  -0.373741  -0.758561
+H   1.680398  -0.373741   0.758561
+}
+
+activate(water)
+set basis aug-cc-pvdz
+set scf_type df
+set dft_radial_points 99
+set dft_spherical_points 302
+
+edft = energy('dldf+d')
+val = -153.80439592484987    #TEST
+compare_values(val, edft, 8, "dlDF+D (2010 +D correction)")  #TEST
+clean()
+
+edft = energy('dldf+d09')
+val = -153.8046388583318276   #TEST
+compare_values(val, edft, 8, "dlDF+D09 (2009 +D correction)")  #TEST
```

Comment:
Are ref `val` coming from psi runs of this input file? If so, the ref value only good to ~6 decimal places (default e_conv for energy), so checking to 8 won't be reliable. If this is the case, need to generate the `val` with e_/d_conv 10 or so, save that to `val`, then set e_/d_conv to 8 in this file, and continue checking to 8.

Reply:
All dldf related lines are copy & paste from the individual test (dft-dldf). Unknown to me when the ref values were generated.

Reply:
Ok, then we'll leave the ref as-is. May have to turn down the number of decimal places in test case in future.